### DISCUSSION TITLE
Adding a FIELD_CHOICES doesn't work correctly.

### DISCUSSION THREAD

**Discussion by damo2929** [netbox-community/netbox#14038] [Help Wanted!] [answered]
### NetBox version

v3.6.3

### Python version

3.10

### Steps to Reproduce

I am trying to add additional status values to vlan

I have added the following to configuration.py
FIELD_CHOICES = {
    'ipam.VLAN.status+': (
      ('planned' , 'Planned' , 'blue')
    )
}

then systemctl restart netbox netbox-rq

### Expected Behavior

expected behaviour is that when opening vlan that the new planned status is on the list.

### Observed Behavior

instead of the new planned status I get 3 lines with | instead 

![image](https://github.com/netbox-community/netbox/assets/43962340/89dbdb24-2373-4317-a10f-2b402fec5f82)

**Comment by DanSheps** [comment, marked as answer]
> FIELD_CHOICES = {
> 'ipam.VLAN.status+': (
> ('planned' , 'Planned' , 'blue')
> )
> }

Try (comma at the end of the choices):

```python
FIELD_CHOICES = {
    'ipam.VLAN.status+': (
        ('planned' , 'Planned' , 'blue'),
    )
}
```

**Reply by damo2929** [reply]
the documentation needs to state that , is required at the end of the selection to stop future questions as I thought it was a typo on the example.

**Reply by candlerb** [reply]
I don't know if this actually works, but IMO it would be clearer if a list of choices were supplied as an actual python _list_ rather than a _tuple_: i.e.

```
FIELD_CHOICES = {
    'dcim.Site.status+': [
        ('fubar', 'FUBAR', 'red'),
    ]
}
```

As well as making it clearer that this is a variable-length list (whereas tuples are normally of fixed size), it also wouldn't matter if the trailing comma is present or not.

The reason the comma is required in the original version is because it's building a tuple containing some inner tuple values, but a single element inside parentheses without a trailing comma is just a value, not a tuple constructor.

```
>>> (0)
0
>>> (0,)
(0,)
>>> (0,1)
(0, 1)
```
